wire-ios/WireUITests/README

```diff
@@ -0,0 +1,6 @@
+Create the env.xcconfig file from by running
+```
+op inject -i wire-ios/WireUITests/env.xcconfig.tpl -o wire-ios/WireUITests/env.xcconfig
```

Comment:
praise: cool, I guess we can add a step in CI `fastlane/Fastfile` to generate the right file